tests/testthat/test-herbarium_label.R

```diff
@@ -56,6 +58,38 @@ test_that("PDF certificates are created", {
 })
 
 
+test_that("PDF certificates are created with an '&' ", {
```

Comment:
What's the difference between this test and the one above? Sorry I can't see it

Reply:
It was supposed to use `data2`, which includes an `& ` in the collector.column, but I think I might have not specified it correctly...